gtk3-wm/fedora-mutter/Dockerfile

```diff
@@ -1,12 +1,11 @@
-ARG FROM_TAG=""
+ARG FROM_TAG="35"
```

Comment:
I'd rather leave it to empty string in order to not forget to update it on each new release

Reply:
Done